zheng-common/src/test/java/com/zheng/common/util/PaginatorTest.java

```diff
@@ -0,0 +1,62 @@
+package com.zheng.common.util;
+
+import static org.mockito.Matchers.anyDouble;
+import com.diffblue.deeptestutils.Reflector;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import java.lang.reflect.InvocationTargetException;
+
+@RunWith(PowerMockRunner.class)
+@PowerMockIgnore({"javax.management.*"})
+public class PaginatorTest {
+
+	/* testedClasses: Paginator */
+	@Test
+	public void initTotalPageOutputVoid() {
```

Comment:
This test provides 100% coverage for the method :-)
I wonder if we could use some of the setters so that the expected value is non-zero?